docker-compose.yml

```
@@ -125,7 +138,8 @@ services:
   st2notifier:
     image: ${ST2_IMAGE_REPO:-stackstorm/}st2notifier:${ST2_VERSION:-3.3.0}
     restart: on-failure
-    depends_on: ["st2api"]
+    depends_on:
+      - st2api
```

```suggestion
    depends_on:
      - redis
      - st2api
```

According to https://docs.stackstorm.com/reference/ha.html#st2notifier

---

Oh, good catch!